Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM ubuntu:latest
```

Comment:
So this only works if you pin this to `ubuntu:trusty`, otherwise the node source setup script fails (on xenial or later).